samples/actions/sam-build.yaml

```diff
@@ -0,0 +1,47 @@
+## Sam Build & Deploy
+
+## Cloning the Repository
+- action: 
+  uses: git-clone
+  with:
+    branch: main
+    repo-name: Product-Management
+    connector: cd-demo
+---
+
+## Download Manifest
+- action: 
+  uses: harness-download
+
+--- 
+
+## SAM Build 
+- action:
+  uses: sam-build
+  with:
+    version: 2.0 ## Optional
+    command: "--use-container" ## Optional
+    build-container-registry: "ecr" ## Optional
+    image-pull-policy: always
+    user: 1000
+    memory: 10mb
+    cpu: 0.25
+    args:
+      - SAM_LOG: "true"
+
+---
+
+## SAM Deploy
+- action:
+  uses: sam-deploy
+  with:
+    version: 2.0 ## Optional
```

Comment:
Is version field really needed for all SAM and Serverless Steps ?

Reply:
@piyush-bhuwalka that show the current plugin is designed, are you saying this field is no longer needed? Im going based of the current yaml for the plugin steps in CD.

Reply:
I have tagged you in a comment in multiple runtime support for serverless enhancement. More details are provided there